alephmarcreader/abstractalephmarcreader.py

```diff
@@ -353,4 +362,117 @@ def get_recipient_organisation(self):
 
         return recipient_organisation
 
-    get_recipient_organisation.__annotations__ = {'return': [Organisation]}
\ No newline at end of file
+    get_recipient_organisation.__annotations__ = {'return': [Organisation]}
+
+    def get_supplement_remarks(self):
+        """
+        Returns comments on supplementing material following the letter. (See also Description.supplement.)
+        :return: [str]
+        """
+        remarks = []
+
+        for field in self.__get_field('525'):
+            footnote_text = self._handle_subfields_cardinality_max_one(self.__get_subfield_texts(field, 'a'))
+
+            if footnote_text:
+                remarks.append(footnote_text)
+
+        return remarks
+
+    get_supplement_remarks.__annotations__ = {'return': [str]}
+
+    def get_document_state(self):
+        """
+        Returns a description of the document state. (i.e. "original", "copy", "draft", etc.)
+        :return: [str]
+        """
+        states = []
+
+        for field in self.__get_field('250'):
+            footnote_text = self._handle_subfields_cardinality_max_one(self.__get_subfield_texts(field, 'a'))
+
+            if footnote_text:
+                states.append(footnote_text)
+
+        return states
+
+    get_document_state.__annotations__ = {'return': [str]}
+
+    def get_original_date_and_place(self):
+        """
+        Returns a transcription of the date and place, as given on the letter.
+        (If it starts with square bracket, it's conjectured.)
+        :return: [OriginalDate]
+        """
+        original_dates = []
+
+        for field in self.__get_field('264'):
+            date = self._handle_subfields_cardinality_max_one(self.__get_subfield_texts(field, 'c'))
+            place = self._handle_subfields_cardinality_max_one(self.__get_subfield_texts(field, 'a'))
+            # TODO Ignore Fields starting wit '['?
+
+            orig_date = self.OriginalDate(date, place)
+            original_dates.append(orig_date)
+
+        return original_dates
+
+    get_original_date_and_place.__annotations__ = {'return': [OriginalDate]}
+
+    def get_references_to_related_entries(self):
+        """
+        Returns a list of related entries.
+        That is usually the case, if there are several document stages of the same letter.
+        (E.g. Original, draft, copy, etc.)
+        :return: [str]
+        """
+        references = []
```

Comment:
So this combines subfields from different fields into one array?

Reply:
Yes.

I could not quite make sense of the Fields 533, 534, 544 (probably they also don't really do make sense in the first place).  
But from what I can tell, the general idea is that different stages of a letter refer to other states from the same letter in different fields. (E.g. the copy would refer to its original in a different field than the original to its copy.)

Sulamith and I aggreed that it would be the easiest to just combine these things on BEOL, as it's basically one type of information, that HAN required to splitt up.  
(Also, currently, these fields are not used systematically. But that should change at some point.)

If you don't like this, feel free to make a suggestion.

Reply:
I think that's ok how it is.

Reply:
Alternatively, I could create a subclass that stores an array for these fields separately, and has a method that returns a combined array of all three fields.  
Might come in handy, if ever we need them separately anyway.

If you like the idea, just assign an issue to me.

Reply:
I think the current implementation is sufficient.